### PR TITLE
[ao] Adding FAQ to docs

### PR DESCRIPTION
**Pull request by HDCharles**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #87322

Summary: migrated from: https://discuss.pytorch.org/t/quantization-frequently-asked-questions/161251

Test Plan: circle CI tests

Reviewers:

Subscribers:

Tasks:

Tags:

### CODE REVIEW

File: docs/source/quantization.rst [hunk 1]
```diff
@@ -998,6 +998,30 @@ if ``dtype`` is ``torch.qint8``, make sure to set a custom ``quant_min`` to be `
...
+
+   ii) Integration with cudnn through native quantized cuda ops: see examples in `pytorch/test_quantized_op.py <https://github.com/pytorch/pytorch/blob/master/test/quantization/core/test_quantized_op.py#L4598>`_
+       this project is an early prototype and has been paused
+
+2. Where can I get ONNX support for my quantized model?:
+
+   Supporting export to onnx model is not a priority for PyTorch quantization, please open an issue in `GitHub - onnx/onnx <https://github.com/onnx/onnx>`_  when you encounter problems with ONNX,
+   or reach out to people in this list: `PyTorch Governance | Maintainers | ONNX exporter <https://pytorch.org/docs/stable/community/persons_of_interest.html#onnx-exporter>`_
+
+3. How can use quantization with LSTM's?:
```
Comment (z-a-f) [inline docs/source/quantization.rst:1019]: nit: Is a word missing? Seems awkward

File: docs/source/quantization.rst [hunk 2]
```diff
@@ -998,6 +998,30 @@ if ``dtype`` is ``torch.qint8``, make sure to set a custom ``quant_min`` to be `
...
+
+   i) PyTorch quantization + fx2trt lowering, inference in TensorRT (A100 and later GPUs): see examples in
+      `pytorch/TensorRT <https://github.com/pytorch/TensorRT/blob/master/py/torch_tensorrt/fx/test/quant/test_quant_trt.py>`_
+
+   ii) Integration with cudnn through native quantized cuda ops: see examples in `pytorch/test_quantized_op.py <https://github.com/pytorch/pytorch/blob/master/test/quantization/core/test_quantized_op.py#L4598>`_
+       this project is an early prototype and has been paused
+
+2. Where can I get ONNX support for my quantized model?:
+
+   Supporting export to onnx model is not a priority for PyTorch quantization, please open an issue in `GitHub - onnx/onnx <https://github.com/onnx/onnx>`_  when you encounter problems with ONNX,
```
Comment (z-a-f) [inline docs/source/quantization.rst:1016]: nit: IMHO, saying that it's not a priority sounds a little dismissive. Could we reword it?

**Review by z-a-f — APPROVED** [review]
I think we can always add more items here, if some questions keep on popping up

File: docs/source/quantization.rst (hunk 1 above)
Comment (HDCharles) [reply docs/source/quantization.rst]: good catch

File: docs/source/quantization.rst (hunk 2 above)
Comment (HDCharles) [reply docs/source/quantization.rst]: yeah, i've changed the language to omit that part

**Pull request closed without merging**